packages/lesswrong/server/utils/feedUtil.ts

```diff
@@ -143,6 +145,7 @@ export async function mergeFeedQueries<SortKeyType>({limit, cutoff, offset, sort
       return subqueryResults.map((result: DbObject) => ({
         type: subquery.type,
         sortKey: subquery.getSortKey(result),
+        isNumericallyPositioned: subquery.isNumericallyPositioned,
```

Comment:
This line fixes a bug that's currently live on production where numerically positioned results aren't actually shown at all (whoops)